Cargo.toml

```diff
@@ -6,8 +6,8 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-llvm-sys = "130"
 scopeguard = "1.1.*"
+inkwell = { git = "https://github.com/TheDan64/inkwell", branch = "master", features = ["llvm13-0"] }
```

Comment:
Might be good to point this at a crates.io release instead, but the most recent preview doesn't have a lot of useful things so I'm locking it to a commit instead